lib/api/src/sys/externals/function.rs

```diff
@@ -713,16 +803,19 @@ impl<'a> Exportable<'a> for Function {
 }
 
 /// Host state for a dynamic function.
+#[cfg(feature = "compiler")]
```

Comment:
We should not require compiler in the function externals anywhere (as the trampolines should be present when running a precompiled module)

Reply:
Changes were reverted.